packages/teams/src/remove-team.ts

```diff
@@ -0,0 +1,23 @@
+import { UserSession } from "@esri/arcgis-rest-auth";
+import { removeGroup, unprotectGroup } from "@esri/arcgis-rest-portal";
+
+/**
+ * removeTeam unprotects a group, then deletes it
+ *
+ * @export
+ * @param {string} id Team Id
+ * @param {UserSession} authentication authentication
+ * @return {*}  {Promise<any>}
```

Comment:
Should methods return an explicit outcome and verification of what was deleted. That would be useful for multiple async calls